### PR TITLE
feat(page-banner): add component - INNO-1230

### PR DESCRIPTION
**Pull request by emeryro**
# PR description

Create new component, based on [specifications](https://webgate.ec.europa.eu/CITnet/confluence/display/NEXTEUROPA/Page+banner+-+Design+specifications)

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 0cd9d5d3cb347065a1357b9845ea1080ad9899b1

https://deploy-preview-894--europa-component-library.netlify.com

**Review by degliwe — CHANGES_REQUESTED** [review]
- Can we make the markup more semantic?
- The wrapper could be  `<section />`
- The title could use a `<h*/>`
- ...

File: src/systems/ec/implementations/react/packages/page-banner/PageBanner.jsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  className,
+  ...props
+}) => {
+  const classNames = classnames(className, 'ecl-page-banner', {
+    [`ecl-page-banner--${variant}`]: variant,
+    [`ecl-page-banner--centered`]: isCentered,
+  });
+
+  return (
+    <section className={classNames} {...props}>
```
Comment (yhuard) [inline src/systems/ec/implementations/react/packages/page-banner/PageBanner.jsx]:
```suggestion
    <section {...props} className={classNames}>
```

File: src/systems/ec/implementations/vanilla/packages/ec-component-page-banner/ec-component-page-banner.scss [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+  }
+
+  .ecl-page-banner__content {
+    display: block;
+    padding: $ecl-spacing-l;
+  }
+
+  .ecl-page-banner__image {
+    background-size: cover;
+    height: (1/4) * 100vw;
```
Comment (yhuard) [inline src/systems/ec/implementations/vanilla/packages/ec-component-page-banner/ec-component-page-banner.scss:49]: yeah, or just `25vw` :)

```suggestion
    height: 25vw;
```

**Review by yhuard — CHANGES_REQUESTED** [review]

File: src/systems/ec/implementations/vanilla/packages/ec-component-page-banner/ec-component-page-banner.scss (hunk 2 above)
Comment (emeryro) [reply src/systems/ec/implementations/vanilla/packages/ec-component-page-banner/ec-component-page-banner.scss:49]: yes, it works too, but I find more explicit this way (we see the ratio directly)

**Review by yhuard — APPROVED** [review]

**Pull request merged**